.github/workflows/deploy.yml

```diff
@@ -45,10 +45,13 @@ jobs:
           AWS_CLOUDFRONT_PUBLIC_KEY: "${{ secrets.AWS_CLOUDFRONT_PUBLIC_KEY_A }}"
           AWS_CLOUDFRONT_PRIVATE_KEY: "${{ secrets.AWS_CLOUDFRONT_PRIVATE_KEY_A }}"
           # AWS_CLOUDFRONT_PUBLIC_KEY_EXPIRING: "${{ secrets.AWS_CLOUDFRONT_PUBLIC_KEY_B }}"
+          BASIC_AUTH_USER: ${{ secrets.BASIC_AUTH_USER }}
```

Comment:
Don't forget to set these values via GitHub UI :)

Reply:
Thanks for the reminder :)